frankenphp/docker-entrypoint.sh

```diff
@@ -26,6 +26,10 @@ if [ "$1" = 'frankenphp' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 		composer install --prefer-dist --no-progress --no-interaction
 	fi
 
+    # Display information about the current project
+	# Or an error about an error in project initialization
```

Comment:
```suggestion
	# Or about an error in project initialization
```

Reply:
Thanks @jdreesen for the suggestion! Applied the changes.

Also noticed and fixed the incorrect indents thanks to this suggestion.